app/api/v2/resource/users.rb

```diff
@@ -34,7 +34,7 @@ def validate_topic!(topic)
         end
         get '/activity/:topic' do
           validate_topic!(params[:topic])
-          data = current_user.activities
+          data = current_user.activities.sort_by {|t| -t.created_at.to_i }
```

Comment:
Better to use `.order('created_at DESC')`, so it will look like

```ruby
data = current_user.activities.order('created_at DESC')
```

Reply:
updated